components/tree/Tree.tsx

```diff
@@ -107,7 +108,7 @@ export interface TreeProps<T extends BasicDataNode = DataNode>
     RcTreeProps<T>,
     'prefixCls' | 'showLine' | 'direction' | 'draggable' | 'icon' | 'switcherIcon'
   > {
-  showLine?: boolean | { showLeafIcon: boolean };
+  showLine?: boolean | { showLeafIcon: boolean; leafIcon?: LeafIcon };
```

Comment:
```suggestion
  showLine?: boolean | { showLeafIcon: boolean | React.ReactNode };
```

How about?

Reply:
Yeah, good idea. Will make the changes & add tests for the util

Reply:
Updated and added some tests for better coverage 😄 FYI @afc163